### ISSUE TITLE
Alerts groups defined in a specific Kubernetes namespace are delayed by 5 minutes

### ISSUE THREAD

**Issue by gurumee92** [grafana/grafana#72427]
### What happened?

Alerts from a group of grafana instances running in a specific Kubernetes namespace are delayed by 5 minutes, despite having the same datasource and alert rule settings. (They work fine in other namespaces).

The alert rule query is as follows
<img width="1252" alt="alert_rule_query" src="https://github.com/grafana/grafana/assets/32067275/64d46415-800a-4982-b965-9b6f1bf43a4e">

If we adjust the query to force an alert, the alert is raised after 5 minutes.

Here is the state history and the time the alert occurred, you can see that the state changed to alerting at 12:55 and the actual message was delivered to Slack at 12:59, 5 minutes later.
<img width="749" alt="증상1-1" src="https://github.com/grafana/grafana/assets/32067275/2f587d2b-37b8-413f-ac87-b575f574957c">
![증상1](https://github.com/grafana/grafana/assets/32067275/8dffdf29-0ece-4313-a613-28f9ce4a7aa4)

Again, we can see that even if we change the alert rule to force a RESOLVED status, the slack message is delayed by more than 5 minutes. (alerting → normal 13:01, receive slack message: 13:09)
<img width="756" alt="증상3-1" src="https://github.com/grafana/grafana/assets/32067275/e39a2f1d-154a-45d6-a396-13b7a0073127"
![증상3](https://github.com/grafana/grafana/assets/32067275/5befaa8e-54c7-4de6-a6e1-a24afaa7e5e8)
>

What's also unusual is that the alarm state changed, and between 13:01-13:09, there was one more alarm that was alerted before it was delivered.
![증상2](https://github.com/grafana/grafana/assets/32067275/fa545de0-9fdf-4fcb-bc92-ac70044577cc)

Could this be an effect of the recent migration from 9.2.5 to 9.4.10, I'd like to know why.

### What did you expect to happen?

Depending on your “alert evaluation behavior”, “timing options” configurations, alerts should be delivered as Slack messages within one minute of the alert instance being created.

**alert evaluation behavior:**

for: 0, every: 1m
<img width="1294" alt="alert_evaluation_behavior" src="https://github.com/grafana/grafana/assets/32067275/0b06c791-0281-4e56-9a05-10ceba173a2d">

<img width="783" alt="evaluation_group" src="https://github.com/grafana/grafana/assets/32067275/cbcddf40-f214-4101-8115-28643fbc5983">

**timing options:**

this following root policy configuration. not overrided

group_wait: 30s, group_interval: 5m, repeat_interval: 4h
<img width="777" alt="timing_options" src="https://github.com/grafana/grafana/assets/32067275/af6f6d23-1139-431b-931f-e040c106f38d">

### Did this work before?

We are not experiencing this issue in our current production environment Grafana (v9.2.5). It is only happening in our beta environment Grafana (v9.4.10), and it is only happening in certain namespaces.

In Kubernetes, we are currently running two instances of Grafana (peering )in multiple namespaces. For our beta environment, we are setting each database to write to the same RDS.

Below is the Grafana database setup that is causing the problem, and the Grafana database setup that is working fine

problem grafana’s configuration:
```bash
[database]
    host = ${GF_DATABASE_HOST}
    name = infra
    password = ${GF_DATABASE_PASSWORD}
    type = mysql
    user = svc_grafana
```

normal grafana’s configuration:
```bash
[database]
    host = ${GF_DATABASE_HOST}
    name = playground
    password = ${GF_DATABASE_PASSWORD}
    type = mysql
    user = svc_grafana
```

### How do we reproduce it?

The issue is not reproducible using the usual methods.

Currently, this issue only occurs on a few grapnas under a specific Kubernetes namespace. Please note the settings above.

### Environment (with versions)?

Grafana: 9.4.10
OS: linux arm64
Browser: chrome v114.0.5735.248(official build) (x86_64) (I guess Does not matter)

### Grafana platform?

Kubernetes

### Datasource(s)?

Prometheus (Thanos)

**Comment by usmangt** [comment]
Hi @gurumee92,

Thanks for opening this issue.

I will forward this to our Engineers so that they can have a look and might be able to tell more details about it.

**Comment by armandgrillet** [comment]
Your `group interval` is set to 5 minutes in your last screenshot, that is coming from the default policy. Change it to 1 minute to receive alerts without delay. 

You could also disable the rule grouping in the notification policy:
<img width="467" alt="SS 2023-07-28 at 10 27 39" src="https://github.com/grafana/grafana/assets/2117580/e309e156-c1ec-4d2f-9c3f-7e498595da65">

**Comment by gurumee92** [comment]
Q1. Can anyone tell me why you are changing it from 5 minutes to 1 minute?

Isn't group_wait the setting that affects when the first alert is raised? 

Q2. Also, why is the grouping setting disabled?

Grafana in another namespace is delivering alerts just fine, even though I'm grouping with grafana_folder, alertname. (Even the timing options setting in the root policy is the same).

**Comment by gurumee92** [comment]
@armandgrillet Thank you for your response. However, I don't understand your answer.

Q1. Can anyone tell me why you are changing it from 5 minutes to 1 minute?

Isn't group_wait the setting that affects when the first alert is raised? 

Q2. Also, why is the grouping setting disabled?

Grafana in another namespace is delivering alerts just fine, even though I'm grouping with grafana_folder, alertname. (Even the timing options setting in the root policy is the same).

**Comment by gurumee92** [comment]
@armandgrillet 
Unfortunately, I tried changing the group_by configuration to Disabled as you said, but the 5 minute delay is still the same. 😭 plz help me (I overrode it in the child notification policy, not the root policy).
